render/render-pokemon.js

```diff
@@ -0,0 +1,15 @@
+
+export { renderPokemon };
+
+function renderPokemon(arr){ // takes in an array
+    //make counter=1
+    let counter = 1;
+    //forEach iterates arr
+    // grab image from dom (render-{counter})
+    // change source of image to arr[counter-1]
+    arr.forEach(() => {
+        const imgEl = document.getElementById(`render-${counter}`);
+        imgEl.src = arr[counter - 1].url_image; //counter starts at 1, so subtract one for arr index
+        counter++;
+    });
```

Comment:
clever iterating through the array! note that you could have done this without the counter like so:
```
arr.forEach((item, i) => {
// forEach takes a second, optional, argument which is the the index of the current iteration
        const imgEl = document.getElementById(`render-${i}`);
        imgEl.src = item.url_image;
```